Recent versions of Boost requires C++17 so we enforce it for the build

--- CMakeLists.txt.orig	2025-03-10 14:47:29.661273709 +0100
+++ CMakeLists.txt	2025-03-10 14:47:39.773327546 +0100
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.7)
 project(REViewer VERSION 0.2.7 LANGUAGES CXX)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 if (NOT CMAKE_BUILD_TYPE)
